tests/system/test_root_metronome.py

```diff
@@ -239,6 +240,26 @@ def test_docker_job():
         assert len(client.get_runs(job_id)) == 1
 
 
+@shakedown.dcos_1_10
+@pytest.mark.skipif("shakedown.ee_version() is None")
+def test_secret_env_var(secret_fixture):
+
+    secret_name, secret_value = secret_fixture
+
+    client = metronome.create_client()
+    job_id = uuid.uuid4().hex
+    job_def = common.job_with_secrets(job_id, secret_name=secret_name)
+    with job(job_def):
+        client.run_job(job_id)
+
+        @retry(wait_fixed=1000, stop_max_delay=5000)
+        def task_exist():
+            assert len(client.get_runs(job_id)) == 1
```

Comment:
please add some meaningful string description for the asserts to ease the debugging when assert fails